### DISCUSSION TITLE
Allow useTranslations in overridden components

### DISCUSSION THREAD

**Discussion by LuHugo** [withastro/starlight#1060] [Feature Requests]
### What version of `starlight` are you using?

0.11.2

### What is your idea?

I'm overriding the pagination component, but the multi-language translation module in it doesn't work because it's an internal module and not exported. Is it possible to export this module or is there any other way to use multi-language module.

### Why is this feature necessary?

Overriding components requires multi-language modules

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
This will be available in our next release — v0.14 🎉 

Overridden components will be able to access a `labels` object in their props that contains all the translated strings in the locale for the current page.

Docs: https://starlight.astro.build/reference/overrides/#labels
